docker/gaffer-gremlin/example/notebooks/graphnotebook-modern-example.ipynb

```diff
@@ -0,0 +1,138 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GafferPop Modern Example Using `graph-notebook`\n",
+    "\n",
+    "This notebook has some examples using Gremlin modern graph but with the [`graph-notebook`](https://github.com/aws/graph-notebook)\n",
+    "plugin enabled. This allows for some visualisation of sub graphs and results\n",
+    "along with extended tooling."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the modern Graph via the gremlinpython client\n",
+    "from gremlin_python.driver.client import Client\n",
+    "\n",
+    "Client('ws://localhost:8182/gremlin', 'g').submit(\"graph.io(graphml()).readGraph('data/tinkerpop-modern.xml')\")"
```

Comment:
I have found it temperamental as well I have been thinking it might be better to load the data via the standard REST API and have the gremlin side read only in the example to be more representative. Ideally we don't want to allow modification via Gremlin anyways as this could cause race conditions if a REST API and gremlin server were pointing to the same accumulo cluster.